docs/man/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright 2011 Free Software Foundation, Inc.
```

Comment:
2020, and unless you want to, you don't have to assign copyright to the FSF anymore (but you'll need to sign-off (`git commit -s`) on your commits)